docker-compose.yml

```diff
@@ -0,0 +1,39 @@
+services:
+  db-mysql:
+    image: mysql:latest
+    restart: unless-stopped
+    env_file: ./.env
+    environment:
+      - MYSQL_DATABASE=$MYSQLDB_DATABASE
+      - MYSQL_USER=$MYSQLDB_USER
+      - MYSQL_PASSWORD=$MYSQLDB_PASSWORD
+      - MYSQL_ROOT_PASSWORD=$MYSQLDB_ROOT_PASSWORD  # Password for root-user. (If I delete it, the containers will stop.)
+                              # I think it's a reason why I received exceptions. Learn about it!
```

Comment:
it's not obvious, but environment variables cannot have an arbitrary name by default. Remove comment